cogent/Makefile

```diff
@@ -64,25 +60,19 @@ doc:
 
 clean:
 	$(E) "Cleaning"
+	$(Q) $(CABAL) clean
+	$(Q) make -C doc clean
+	$(Q) find . -name "*_flymake.hs" -delete
+
+full-clean: clean
 	$(Q) $(CABAL) sandbox delete -v0
 	$(Q) rm -rf .cabal-sandbox/ dist/ ../isa-parser/dist
 	$(Q) rm -rf out/
 	$(Q) rm -rf tests/include
-	$(Q) make -C doc clean
 	$(Q) find . -name "*_flymake.hs" -delete
 
-install-global: cabal-update isa-parser .install-deps configure build .install
-	$(E) "Cogent Compiler successfully built for $(OS)."
-	$(E)
-	$(E) "Add 'cogent' compiler to your path by running:"
-	$(E) '  export PATH=$$PATH:~/.cabal/bin'
-	$(E)
-	$(E) "To enable bash auto-completion for 'cogent', please add the following to your .bashrc file:"
-	$(E) "source $(PWD)/misc/cogent_autocomplete.sh"
-	$(E)
-	$(E)
 
-install-dev: sandbox .install-deps configure build .install
+install-dev: cabal-update sandbox .install-deps configure build .install
```

Comment:
I'm planning to get rid of the 'global' install option altogether. Are you ok with that?